src/ContentBundle/Controller/Backend/ContentEditorController.php

```diff
@@ -33,11 +33,22 @@ public function designAction($owner, $ownerId)
     {
         /** @var BlockManager $blockManager */
         $blockManager = $this->get('opifer.content.block_manager');
+        $manager = $this->get('opifer.content.content_manager');
 
         /** @var AbstractDesignSuite $suite */
         $suite = $this->get(sprintf('opifer.content.%s_design_suite', $owner));
         $suite->load($ownerId);
 
+        $content = $manager->getRepository()->find($ownerId);
+
+        $defaultDomain = null;
+
+        if($content->getSite()->getDefaultDomain()){
+            $defaultDomain = $content->getSite()->getDefaultDomain()->getDomain();
+        } elseif ($content->getSite()->getDomains()) {
+            $defaultDomain = $content->getSite()->getDomains()->first()->getDomain();
+        }
```

Comment:
I think it's better to implement this functionality inside `Site::getDefaultDomain`